protos/bc.proto

```diff
@@ -24,24 +24,32 @@ message RpcTransaction {
 message RpcFeedTransaction {
     string owner_addr = 1;
     string feed_addr = 2;
-    string amount = 3;
-    string tx_fee = 4;
-    string private_key_hex = 5;
+    string data_type = 3;
+    string data_length = 4;
+    string data = 5;
```

Comment:
What form the data will have? Will it always be just UTF-8 text? If not, `bytes` type would be more appropriate